### DISCUSSION TITLE
Detect Repeated Crashing and Stop Injecting

### DISCUSSION THREAD

**Discussion by shellster** [valinet/ExplorerPatcher#1351] [Ideas]
I really like ExplorerPatcher, but the solution is obviously brittle, especially when you shut down mid-windows update, and relocated to a place without an internet connection and finish the Windows Update.  On more than one occasion I've reached a place where explorer.exe is crashing over and over again because of ExplorerPatcher not being able to get latest symbols (either no internet or I can't easily configure wireless with explorer.exe crashing constantly and stealing focus).  I would really appreciate if ExplorerPatcher could detect crashing (or even just detect that it had tried to load so many times in the last 10 seconds) and just stop injecting itself on repeated failure until the next windows launch (or after one successful explorer.exe launch without it) as the current non-stop attempts make the system nearly unusable or recoverable without booting into safe-mode.

**Comment by topia** [comment]
Already implemented.
https://github.com/valinet/ExplorerPatcher/commit/d7e5b7d3c965803d4d52aefd7fedefe11e3518c4

**Reply by shellster** [reply]
Thanks I missed this!
